tools/builds/make.bat

```diff
@@ -1 +1,5 @@
-cmake -G "Visual Studio 12" -D CMAKE_INSTALL_PREFIX=install -D CMAKE_BUILD_TYPE=Release "--build build" .
+set BUILDDIR=%1
+IF [%BUILDDIR%] == [] (
+    set BUILDDIR=.
+)
```

Comment:
Thinking about it, builddir is probably the wrong name since it's supposed to point to the source (build the thing in this directory, not build the thing using this directory)